iis-common/src/main/java/eu/dnetlib/iis/common/WorkflowRuntimeParameters.java

```diff
@@ -29,6 +29,20 @@ public static String getParamValue(String paramName, Configuration configuration
     }
 	
     /**
+     * Retrieves {@link Integer} parameter from hadoop context configuration when set to non-empty value different than {@link WorkflowRuntimeParameters#UNDEFINED_NONEMPTY_VALUE}.
+     * Null is returned when parameter was not set.
+     * @throws {@link NumberFormatException} if parameter value does not contain a parsable integer
+     */
+    public static Integer getIntegerParamValue(String paramName, Configuration configuration) throws NumberFormatException {
+        String paramValue = configuration.get(paramName);
+        if (StringUtils.isNotBlank(paramValue) && !UNDEFINED_NONEMPTY_VALUE.equals(paramValue)) {
+            return Integer.parseInt(paramValue);
```

Comment:
> Integer.valueOf

We do have autoboxing and what `Integer.valueOf()` does is calling `Integer.parseInt()`. 

I've fixed that too since I have refactored this piece of code anyway. Maybe it's better to have both return types aligned...